orchestrator/src/main/java/com/scottlogic/datahelix/generator/orchestrator/generate/GenerateCommandLine.java

```diff
@@ -46,6 +47,7 @@
     descriptionHeading = "%nDescription:%n",
     parameterListHeading = "%nParameters:%n",
     optionListHeading = "%nOptions:%n",
+    version = { "Profile Schema Version " + ProfileConfiguration.PROFILE_SCHEMA_VERSION},
     abbreviateSynopsis = true)
```

Comment:
We have two versions, the schema version and the datahelix version. How should this be displayed?